includes/class-rop-activator.php

```diff
@@ -44,6 +44,35 @@ public static function activate() {
 			add_option( 'rop_first_install_version', $version );
 		}
 
+		$logger = new Rop_Logger();
+
+		// Get unique token
+		if ( ! class_exists( '\GuzzleHttp\Client' ) ) {
+			$logger->alert_error( 'Error: Cannot find Guzzle' );
+			return;
+		}
+
+		$client = new GuzzleHttp\Client();
+
+		try {
+			$app_url = ROP_AUTH_APP_URL . ROP_APP_ACTIVATION_PATH;
+			$current_url = ( isset( $_SERVER['HTTPS'] ) && $_SERVER['HTTPS'] === 'on' ? 'https' : 'http' ) . '://' . $_SERVER[ HTTP_HOST ];
```

Comment:
For websites using cloudflare universal SSL this test would not be accurate, please also check for the following so we cover those cases: 

```!empty( $_SERVER['HTTP_X_FORWARDED_PROTO'] ) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https' ```

Reply:
went ahead and fixed this